pkg/asset/store.go

```diff
@@ -26,19 +26,27 @@ type Store interface {
 	Destroy(Asset) error
 }
 
-type onDiskAssetState struct {
-	asset            WritableAsset
-	present          bool
-	matchesStateFile bool
-	anyParentsDirty  bool
+type assetSource string
+
+const (
```

Comment:
do you think enums are better suited for this?

Reply:
Yes. I've changed it.